go/internal/substreams/ethereum/handlers/nft.go

```diff
@@ -20,6 +21,28 @@ type TransferInput struct {
 	TokenID *big.Int       `json:"tokenId"`
 }
 
+func (h *Handler) handleTransfer(tx *pb.Tx, nftContract string, from string, to string, tokenID int64) error {
+	var collection indexerdb.TeritoriCollection
+	if err := h.indexerDB.Where("nft_contract_address = ? AND network_id = ?", nftContract, h.network.ID).First(&collection).Error; err != nil {
+		return errors.Wrap(err, "failed to get collection")
+	}
+
+	nftId := h.network.NFTID(collection.MintContractAddress, fmt.Sprintf("%d", tokenID))
+	var nft indexerdb.NFT
+	if err := h.indexerDB.Where("id", nftId).First(&nft).Error; err != nil {
+		return errors.Wrap(err, "failed to get nft")
+	}
+
+	nft.OwnerID = h.network.UserID(to)
+	if err := h.indexerDB.Save(&nft).Error; err != nil {
+		return errors.Wrap(err, "failed to update nft owner")
+	}
+
+	// NOTE: This is used only to handle transfers made by another contract so we dont track activities
```

Comment:
why? in the teritori indexer, we still add to activity in this case (transfer activity)
can be transfer to contract or user, make sense to add it to activity